docs/Change_Log.md

```diff
@@ -1,5 +1,18 @@
 # Change Log
 
+## API v9
+
+#### April 28, 2021
+
+API v9 is now available.
+
+API v9 includes supports for [threads](#DOCS_TOPICS_THREADS), an upcoming feature.  Older API versions will not receive any Gateway Events for threads, so it is important to update soon!  We've prepared a [migration guide](#DOCS_TOPICS_THREADS) to help make the upgrade process very straightforward.
```

Comment:
Doesn't v9 also need to be added to the Gateway versions table (https://github.com/discord/discord-api-docs/blob/threads/docs/topics/Gateway.md#gateway-versions)?